include/hlslib/xilinx/OpenCL.h

```diff
@@ -28,10 +28,59 @@ namespace ocl {
 
 #if HLSLIB_LEGACY_SDX == 0
 constexpr auto kXilinxMemPointer = CL_MEM_EXT_PTR_XILINX;
-constexpr auto kMemoryBank0 = XCL_MEM_DDR_BANK0;
-constexpr auto kMemoryBank1 = XCL_MEM_DDR_BANK1;
-constexpr auto kMemoryBank2 = XCL_MEM_DDR_BANK2;
-constexpr auto kMemoryBank3 = XCL_MEM_DDR_BANK3;
+
+//constexpr auto kMemoryBank0 = 32 | XCL_MEM_TOPOLOGY;
+//constexpr auto kMemoryBank1 = 33 | XCL_MEM_TOPOLOGY;
+
+class XilinxDDRBankFlags {
+public:
+  XilinxDDRBankFlags(const std::string &device_name) {
+    if(device_name == "xilinx_u280_xdma_201920_3") {
+      _kMemoryBank0 = XCL_MEM_TOPOLOGY | 32;
+      _kMemoryBank1 = XCL_MEM_TOPOLOGY | 33;
+      _kMemoryBank2 = -1;
+      _kMemoryBank3 = -1;
+    }
+    else {
+      defaultAssignment();
+    }
+  }
+
+  XilinxDDRBankFlags() {
+    defaultAssignment();
+  }
+
+  inline int kMemoryBank0() {
+    return _kMemoryBank0;
+  }
+
+  inline int kMemoryBank1() {
+    return _kMemoryBank1;
+  }
+
+  inline int kMemoryBank2() {
+    return _kMemoryBank2;
+  }
+
+  inline int kMemoryBank3() {
+    return _kMemoryBank3;
+  }
+
+private:
+  void defaultAssignment() {
+    _kMemoryBank0 = XCL_MEM_DDR_BANK0;
+    _kMemoryBank1 = XCL_MEM_DDR_BANK1;
+    _kMemoryBank2 = XCL_MEM_DDR_BANK2;
+    _kMemoryBank3 = XCL_MEM_DDR_BANK3;
```

Comment:
Follow naming convention for `FunctionNames` and `variableNames`. These should be `DefaultAssignment` and `memory_bank_0`.